modules/auxiliary/gather/browser_getprivateip.rb

```diff
@@ -0,0 +1,142 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'           => "Private IP Leakage to WebPage using WebRTC Function.",
+        'Description'    => %q(
+          This module exploits a vulnerability in browsers using well-known property of WebRTC (Web Real-Time Communications) which enables Web applications and sites to capture or exchange arbitrary data between browsers without requiring an intermediary.
+        ),
+        'License'        => MSF_LICENSE,
+        'Author'         => [
+          'Brendan Coles', #MSF Module
```

Comment:
Thanks, but I did nothing. I literally grabbed the first chunk of JS I found on GitHub and slapped it in the module.